examples/diff.c

```diff
@@ -335,3 +350,60 @@ static void diff_print_stats(git_diff *diff, struct opts *o)
 	git_buf_free(&b);
 	git_diff_stats_free(stats);
 }
+
+int string_to_int(const char ** out)
```

Comment:
This should just be `const char *` and if it's an input parameter, it definitely shouldn't be named `out`.